server/controllers/temporary-accommodation/manage/premisesController.ts

```diff
@@ -112,7 +112,7 @@ export default class PremisesController {
 
       const updatePremises: UpdatePremises = {
         characteristicIds: [],
-        turnaroundWorkingDayCount: config.flags.turnaroundsDisabled
+        turnaroundWorkingDayCount: !config.flags.turnaroundsDisabled
           ? parseNaturalNumber(req.body.turnaroundWorkingDayCount)
           : null,
         ...req.body,
```

Comment:
This should go above `turnaroundWorkingDayCount`, otherwise our newly parsed `turnaroundWorkingDayCount` will get overwritten by the string `turnaroundWorkingDayCount` on the request body

Reply:
good spot, thanks